packages/create-app/src/lib/tasks.ts

```diff
@@ -236,3 +236,34 @@ export async function moveAppTask(
       });
   });
 }
+
+/**
+ * Initializes a git repository in the destination folder
+ *
+ * @param dir - source path to initialize git repository in
+ * @param context - template parameters
+ * @throws if `exec` fails
+ */
+export async function initGitRepository(dir: string, context: any) {
+  const runCmd = async (cmd: string) => {
+    let cmdResponse = { stdout: '', stderr: '' };
+    await Task.forItem('executing', cmd, async () => {
+      process.chdir(dir);
+      cmdResponse = await exec(cmd).catch(error => {
+        process.stdout.write(error.stderr);
+        process.stdout.write(error.stdout);
+        throw new Error(`Could not execute command ${chalk.cyan(cmd)}`);
+      });
+    });
+    return cmdResponse;
+  };
```

Comment:
I'm thinking of transforming `runCmd` into a reusable function since on `buildAppTask` we have kind of the same function, the difference is that this one returns the command output. Does it make sense?

Anyway, I'll take a look at it tomorrow, I'm done for today 😉

Reply:
Tbh I think it's fine to keep these duplicated. I would even say that we could wrap this entire thing into a single `Task.forItem('init git', ...)` or something like that. I think it's a bit verbose to show the full commands that we run to do that